internal/registry/url_prefixes.go

```diff
@@ -0,0 +1,10 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package registry
+
+const (
+	// SearchAPI is the endpoint for filtering package registry packages
+	SearchAPI = "/search"
```

Comment:
Should this endpoint be exposed to other packages?

Reply:
I was just following how the kibana package was structured, but I'll keep it from being exported